scripts/transpilation/transform-js-scss-css-imports.js

```diff
@@ -46,7 +46,7 @@ console.log('Crunching import paths...');
 // eslint-disable-next-line no-console
 console.log('');
 
-const findReplaces = [{ find: '.module.scss', replace: '.module.css' }];
+const findReplaces = [{ find: '.scss', replace: '.css' }];
```

Comment:
Changing this here rather than renaming the files in `bpk-stylesheets` as that would have created a breaking change folks would need to do if they were using the sass stylesheets over the js